### PR TITLE
[REACTOS] Correct some typos in comments

### PR DESCRIPTION
**Pull request by diakin**
## Purpose

Some typo in comments corrected.. 

JIRA issue: none

## Proposed changes

Some typo in comments corrected.. 4085 files from 14931

### CODE REVIEW

File: dll/3rdparty/libpng/pngpriv.h [hunk 1]
```diff
@@ -1910,7 +1910,7 @@ PNG_INTERNAL_FUNCTION(void,png_ascii_from_fixed,(png_const_structrp png_ptr,
  */
 #define PNG_FP_INVALID  512  /* Available for callers as a distinct value */
 
-/* Result codes for the parser (boolean - true meants ok, false means
+/* Result codes for the parser (boolean - true means ok, false means
```
Comment (binarymaster) [inline dll/3rdparty/libpng/pngpriv.h:1913]: This is 3rd-party code - send changes to upstream.

File: dll/3rdparty/libtirpc/tirpc/reentrant.h [hunk 2]
```diff
@@ -56,7 +56,7 @@
  * One approach for thread safety is to provide discrete versions of the
  * library: one thread safe, the other not.  The disadvantage of this is
  * that libc is rather large, and two copies of a library which are 99%+
- * identical is not an efficent use of resources.
+ * identical is not an efficient use of resources.
```
Comment (binarymaster) [inline dll/3rdparty/libtirpc/tirpc/reentrant.h:59]: This is 3rd-party code - send changes to upstream.

File: dll/3rdparty/libpng/pngpriv.h [hunk 3]
```diff
@@ -316,7 +316,7 @@
 
 /* See pngconf.h for more details: the builder of the library may set this on
  * the command line to the right thing for the specific compilation system or it
- * may be automagically set above (at present we know of no system where it does
+ * may be automatically set above (at present we know of no system where it does
```
Comment (binarymaster) [inline dll/3rdparty/libpng/pngpriv.h:319]: This is not a typo.

File: drivers/network/dd/netkvm/Common/IONetDescriptor.h [hunk 4]
```diff
@@ -118,7 +118,7 @@ struct virtio_net_ctrl_mac {
  * Control VLAN filtering
  *
  * The VLAN filter table is controlled via a simple ADD/DEL interface.
- * VLAN IDs not added may be filterd by the hypervisor.  Del is the
+ * VLAN IDs not added may be filtered by the hypervisor.  Del is the
```
Comment (ThFabba) [inline drivers/network/dd/netkvm/Common/IONetDescriptor.h:121]: Third party, though I can't find the upstream file right now...

File: drivers/network/dd/pcnet/pcnethw.h [hunk 5]
```diff
@@ -63,7 +63,7 @@
 #define CSR0   0x0      /* controller status register */
 #define CSR1   0x1      /* init block address 0 */
 #define CSR2   0x2      /* init block address 1 */
-#define CSR3   0x3      /* interrupt masks and deferral control */
+#define CSR3   0x3      /* interrupt masks and referral control */
```
Comment (ThFabba) [inline drivers/network/dd/pcnet/pcnethw.h:66]: Not a typo

File: drivers/storage/class/cdrom/cdrom.h [hunk 6]
```diff
@@ -135,7 +135,7 @@ extern CDROM_SCAN_FOR_SPECIAL_INFO CdromHackItems[];
 #define MEDIA_CHANGE_TIMEOUT_TIME  300
 
 // Indicates whether is is safe to send StartUnit commands
-// to this device. It will only be off for some removeable devices.
+// to this device. It will only be off for some removable devices.
```
Comment (ThFabba) [inline drivers/storage/class/cdrom/cdrom.h:138]: Third party: https://github.com/microsoft/Windows-driver-samples/blob/main/storage/class/cdrom/src/cdrom.h#L124

File: drivers/storage/class/classpnp/classp.h [hunk 7]
```diff
@@ -246,7 +246,7 @@ extern CONST LARGE_INTEGER Magic10000;
 #define SHIFT10000               13
 
 //
-// Constant to help wih various time conversions
+// Constant to help with various time conversions
```
Comment (ThFabba) [inline drivers/storage/class/classpnp/classp.h:249]: Third party: https://github.com/microsoft/Windows-driver-samples/blob/main/storage/class/classpnp/src/classp.h#L237

File: drivers/storage/class/disk/disk.h [hunk 8]
```diff
@@ -306,7 +306,7 @@ typedef struct _DISK_DATA {
 
     //
     // ReadyStatus - STATUS_SUCCESS indicates that the drive is ready for
-    // use.  Any error status is to be returned as an explaination for why
+    // use.  Any error status is to be returned as an explanation for why
```
Comment (ThFabba) [inline drivers/storage/class/disk/disk.h:309]: Third party: https://github.com/microsoft/Windows-driver-samples/blob/main/storage/class/disk/src/disk.h#L297

File: drivers/storage/ide/uniata/atapi.h [hunk 9]
```diff
@@ -109,7 +109,7 @@ DbgPrint(
 #define PRINT_PREFIX
 
 // Note, that using DbgPrint on raised IRQL will crash w2k
-// ttis will not happen immediately, so we shall see some logs
+// this will not happen immediately, so we shall see some logs
```
Comment (ThFabba) [inline drivers/storage/ide/uniata/atapi.h:112]: Third party: https://alter.org.ua/en/soft/win/uni_ata/

File: drivers/wdm/audio/drivers/ac97/common.h [hunk 10]
```diff
@@ -137,7 +137,7 @@ class CAC97AdapterCommon : public IAC97AdapterCommon,
     NTSTATUS SetAC97Default (void);
 
     //
-    // Aquires the semaphore for AC97 register access.
+    // Acquires the semaphore for AC97 register access.
```
Comment (ThFabba) [inline drivers/wdm/audio/drivers/ac97/common.h:140]: Third party (somewhere from https://github.com/microsoftarchive/msdn-code-gallery-microsoft according to the commit log... so maybe we can consider this a fork?)

File: modules/rosapps/applications/explorer-old/shell/webchild.h [hunk 11]
```diff
@@ -717,7 +717,7 @@ struct ANSUNC DWebBrowserEvents2Impl
 			_callback->DownloadComplete();
 			break;
 
-		  case DISPID_ONTHEATERMODE:	// sent when theater mode should be on/off
+		  case DISPID_ONTHEATERMODE:	// sent when heater mode should be on/off
```
Comment (ThFabba) [inline modules/rosapps/applications/explorer-old/shell/webchild.h:720]: Not a typo

File: modules/rosapps/applications/sysutils/fontsub/resource.h [hunk 12]
```diff
@@ -1,7 +1,7 @@
 /* FontSub by Katayama Hirofumi MZ
  *
  * To the extent possible under law, the person who associated CC0 with
- * FontSub has waived all copyright and related or neighboring rights
+ * FontSub has waived all copyright and related or neighbouring rights
```
Comment (ThFabba) [inline modules/rosapps/applications/sysutils/fontsub/resource.h:4]: Not a typo, valid AE spelling

File: modules/rosapps/drivers/vfd/vfddrv.h [hunk 13]
```diff
@@ -38,7 +38,7 @@ extern "C" {
 //
 typedef enum  _DEVICE_STATE
 {
-	VFD_STOPPED,				// Dvice stopped
+	VFD_STOPPED,				// Device stopped
```
Comment (ThFabba) [inline modules/rosapps/drivers/vfd/vfddrv.h:41]: File header looks like third party, not sure if we consider this forked...

File: ntoskrnl/include/internal/ke_x.h [hunk 14]
```diff
@@ -996,7 +996,7 @@ KiComputeDueTime(IN PKTIMER Timer,
 
 //
 // Called from Unlink and Queue Insert Code.
-// Also called by timer code when canceling an inserted timer.
+// Also called by timer code when cancelling an inserted timer.
```
Comment (ThFabba) [inline ntoskrnl/include/internal/ke_x.h:999]: Valid AE spelling

File: sdk/include/c++/stlport/stl/_alloc.h [hunk 15]
```diff
@@ -515,8 +515,8 @@ class _STLP_alloc_proxy : public _MaybeReboundAlloc {
 
   /* We need to define the following swap implementation for allocator with state
    * as those allocators might have implement a special swap function to correctly
-   * move datas from an instance to the oher, _STLP_alloc_proxy should not break
```
Comment (ThFabba) [inline sdk/include/c++/stlport/stl/_alloc.h:518]: Third party: http://www.stlport.org/

File: sdk/include/ddk/csq.h [hunk 16]
```diff
@@ -197,7 +197,7 @@ typedef IO_CSQ_RELEASE_LOCK *PIO_CSQ_RELEASE_LOCK;
 
 /*
  * Finally, this is called by the queue library when it wants to complete
- * a canceled IRP.
+ * a cancelled IRP.
```
Comment (ThFabba) [inline sdk/include/ddk/csq.h:200]: Valid AE spelling

File: sdk/include/psdk/intsafe.h [hunk 17]
```diff
@@ -32,7 +32,7 @@
 
 /* Handle ntintsafe here too */
 #ifdef _NTINTSAFE_H_INCLUDED_
-#ifndef _NTDEF_ /* Guard agains redefinition from ntstatus.h */
+#ifndef _NTDEF_ /* Guard again redefinition from ntstatus.h */
```
Comment (ThFabba) [inline sdk/include/psdk/intsafe.h:35]: Should be `against`

File: sdk/include/psdk/ms_sal.h [hunk 18]
```diff
@@ -136,7 +136,7 @@
      _Success_(return != FALSE) BOOL
      PathCanonicalizeA(_Out_writes_(MAX_PATH) LPSTR pszBuf, LPCSTR pszPath) :
         pszBuf is only guaranteed to be NULL-terminated when TRUE is returned,
-        and FALSE indiates failure. In common practice, callers check for zero
+        and FALSE indicates failure. In common practice, callers check for zero
```
Comment (ThFabba) [inline sdk/include/psdk/ms_sal.h:139]: Third party -- URL is at the top of the file

File: sdk/include/reactos/drivers/directx/dxg.h [hunk 19]
```diff
@@ -87,10 +87,10 @@
 
 /***********************************************************************************/
 /* This driver functions are exported raw from Eng* it only exists in the def file */
-/* you can not do syscallback to thuse but you can import them from win32k.sys     */
+/* you can not do syscallback to this but you can import them from win32k.sys     */
```
Comment (ThFabba) [inline sdk/include/reactos/drivers/directx/dxg.h:90]: Should be `these` (and so should the "This" in the line above)

File: sdk/include/reactos/libs/expat/expat.h [hunk 20]
```diff
@@ -235,8 +235,8 @@ XML_ParserCreate_MM(const XML_Char *encoding,
                     const XML_Char *namespaceSeparator);
 
 /* Prepare a parser object to be re-used.  This is particularly
-   valuable when memory allocation overhead is disproportionatly high,
-   such as when a large number of small documnents need to be parsed.
+   valuable when memory allocation overhead is disproportionately high,
```
Comment (ThFabba) [inline sdk/include/reactos/libs/expat/expat.h:238]: Third party

File: sdk/include/reactos/libs/fullfat/ff_config.h [hunk 21]
```diff
@@ -67,7 +67,7 @@
 
 
 //---------- FAT12 SUPPORT
-#define FF_FAT12_SUPPORT				// Enable FAT12 Suppport. You can reduce the code-size by commenting this out.
+#define FF_FAT12_SUPPORT				// Enable FAT12 Support. You can reduce the code-size by commenting this out.
```
Comment (ThFabba) [inline sdk/include/reactos/libs/fullfat/ff_config.h:70]: Third party

File: sdk/include/reactos/libs/libmpg123/config.h [hunk 22]
```diff
@@ -29,10 +29,10 @@
 /* The default audio output module(s) to use */
 #define DEFAULT_OUTPUT_MODULE "win32"
 
-/* Define if building with dynamcally linked libmpg123 */
+/* Define if building with dynamically linked libmpg123 */
```
Comment (ThFabba) [inline sdk/include/reactos/libs/libmpg123/config.h:32]: Third party: https://www.mpg123.de/

File: sdk/include/reactos/libs/libtiff/tiffconf.h [hunk 23]
```diff
@@ -103,7 +103,7 @@
 /* Support NeXT 2-bit RLE algorithm */
 #define NEXT_SUPPORT 1
 
-/* Support Old JPEG compresson (read contrib/ojpeg/README first! Compilation
+/* Support Old JPEG compression (read contrib/ojpeg/README first! Compilation
```
Comment (ThFabba) [inline sdk/include/reactos/libs/libtiff/tiffconf.h:106]: Third party

File: sdk/include/reactos/libs/libxslt/numbersInternals.h [hunk 24]
```diff
@@ -63,7 +63,7 @@ struct _xsltFormatNumberInfo {
     int	    group;		/* Number of chars per display 'group' */
     int     multiplier;		/* Scaling for percent or permille */
     char    add_decimal;	/* Flag for whether decimal point appears in pattern */
-    char    is_multiplier_set;	/* Flag to catch multiple occurences of percent/permille */
+    char    is_multiplier_set;	/* Flag to catch multiple occurrences of percent/permille */
```
Comment (ThFabba) [inline sdk/include/reactos/libs/libxslt/numbersInternals.h:66]: Third party

File: sdk/include/xdk/fsrtlfuncs.h [hunk 25]
```diff
@@ -164,8 +164,8 @@ FsRtlUninitializeFileLock(
         default: IofCompleteRequest with STATUS_INVALID_DEVICE_REQUEST;
                  return STATUS_INVALID_DEVICE_REQUEST;
 
-    -'AllwaysZero' is passed thru as 'AllwaysZero' to lock / unlock routines.
-    -'Irp' is passet thru as 'Irp' to FsRtlPrivateLock.
+    -'AllwaysZero' is passed throught as 'AllwaysZero' to lock / unlock routines.
```
Comment (ThFabba) [inline sdk/include/xdk/fsrtlfuncs.h:167]: Should be `through` (but thru is a common colloquial spelling)

File: sdk/lib/fslib/ext2lib/ext2_fs.h [hunk 26]
```diff
@@ -3,7 +3,7 @@
  *
  * Copyright (C) 1992, 1993, 1994, 1995
  * Remy Card (card@masi.ibp.fr)
- * Laboratoire MASI - Institut Blaise Pascal
+ * Laboratoire MASI - Institute Blaise Pascal
```
Comment (ThFabba) [inline sdk/lib/fslib/ext2lib/ext2_fs.h:6]: As above, not a typo

**Review by ThFabba — COMMENTED** [review]
You want to be careful with large-scale changes like this, since our repo contains a lot of third-party code, which we don't want to get out of sync just for typos (the changes would just get reverted next time we sync with upstream). You may find `media/doc/3rd Party Files.txt` useful, although it's not 100% complete -- sometimes file headers give the only hint that something is third-party.

File: dll/3rdparty/libpng/pngpriv.h (hunk 1 above)
Comment (diakin) [reply dll/3rdparty/libpng/pngpriv.h:1913]: OK )

File: dll/3rdparty/libpng/pngpriv.h (hunk 3 above)
Comment (diakin) [reply dll/3rdparty/libpng/pngpriv.h:319]: I suspected it.

File: base/applications/mstsc/types.h [hunk 27]
```diff
@@ -131,7 +131,7 @@ typedef struct _key_translation
 	/* For normal scancode translations */
 	uint8 scancode;
 	uint16 modifiers;
-	/* For sequences. If keysym is nonzero, the fields above are not used. */
+	/* For sequences. If keysum is nonzero, the fields above are not used. */
```
[the remark this answers is not in the document]
Comment (diakin) [reply base/applications/mstsc/types.h:134]: OK

File: boot/environ/include/efi/UgaDraw.h [hunk 28]
```diff
@@ -85,7 +85,7 @@ typedef union {
 } EFI_UGA_PIXEL_UNION;
 
 ///
-/// Enumration value for actions of Blt operations.
+/// Enumeration value for actions of Blt operations.
```
Comment (HBelusca) [inline boot/environ/include/efi/UgaDraw.h:88]: Note that all these headers under include/efi are 3rd-party, so these changes should be either reverted, or somehow sent upstream...

File: drivers/filesystems/ext2/inc/ext2fs.h [hunk 29]
```diff
@@ -340,7 +340,7 @@ Ext2ClearFlag(PULONG Flags, ULONG FlagBit)
 #define Ext2BugCheck(A,B,C,D) { KeBugCheckEx(EXT2_FILE_SYSTEM, A | __LINE__, B, C, D ); }
 
 
-/* Ext2 file system definions *******************************************/
+/* Ext2 file system definitions *******************************************/
```
Comment (binarymaster) [inline drivers/filesystems/ext2/inc/ext2fs.h:343]: 3rd-party code.

**Comment by binarymaster** [comment]
I think one of the commits here reverts too much, including valid typo fixes in non-3rd-party code.

**Pull request closed without merging**